ta/pkcs11/src/processing.h

```diff
@@ -54,4 +54,19 @@ enum pkcs11_rc step_symm_operation(struct pkcs11_session *session,
 
 enum pkcs11_rc tee_init_ctr_operation(struct active_processing *processing,
 				      void *proc_params, size_t params_size);
+
+/*  Digest operations util */
+bool processing_is_tee_digest(uint32_t proc_id);
```

Comment:
prefer `mechanism_is_tee_digest(enum pkcs11_mechanism_id mecha_id)` 
(both function and argument labels)

Reply:
OK